include/boost/geometry/algorithms/detail/is_valid/multipolygon.hpp

```diff
@@ -257,29 +257,30 @@ class is_valid_multipolygon
 
 
     template <typename VisitPolicy, typename Strategy>
-    struct per_polygon
+    struct is_invalid_polygon
```

Comment:
Other places could be changed like this. Now it's clear what's the purpose of this struct.